tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -126,35 +120,16 @@ describe('IssueList', function () {
     });
 
     TagStore.init();
-
-    props = {
-      api,
-      savedSearchLoading: false,
-      savedSearches: [savedSearch],
-      useOrgSavedSearches: true,
-      selection: {
-        projects: [parseInt(organization.projects[0].id, 10)],
-        environments: [],
-        datetime: {period: '14d'},
-      },
-      location: {query: {query: 'is:unresolved'}, search: 'query=is:unresolved'},
-      params: {orgId: organization.slug},
-      organization,
-      tags: tags.reduce((acc, tag) => {
-        acc[tag.key] = tag;
-
-        return acc;
-      }),
-    };
   });
 
   afterEach(function () {
+    cleanup();
     jest.clearAllMocks();
     MockApiClient.clearMockResponses();
-    if (wrapper) {
-      wrapper.unmount();
+
+    for (const el of document.querySelectorAll('style').values()) {
```

Comment:
not sure if we need this @billyvg

Reply:
Ah yeah I have some perf related experiments in here. These don't seem to amount to much.